Dockerfile

```diff
@@ -0,0 +1,8 @@
+# pull official base image
+FROM python:3.8.6
+
+RUN apt-get install libsndfile1
+
+RUN pip install --no-cache datasets transformers soundfile numpy torch torchaudio
+
+COPY . .
```

Comment:
Can you add `RUN pip install -e .` after this line?

Reply:
Oh yeah. Sure thin. I also removed the README from the .dockerignore as it is embedded in the setup.

Reply:
> Oh yeah. Sure thing. I also removed the README from the .dockerignore as it is embedded in the setup.

It is before `COPY . .`. I think it should run `RUN pip install -e .` after `COPY . .`.

Reply:
Oh I'm so sorry. Must be getting dizzy 🥴. I just tried it in the already existing test container on my machine and forgot that the relevant files sure need to be copied.😵‍💫 Should be fixed now.